src/components/Modal.js

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 import '../styles/Modal.css';
 
 const Modal = props => {
+  const { children, setDisplay } = props;
+
   return (
     <div className="modal-wrapper">
       <div className="modal-backdrop" />
       <div className="modal-box">
-        <span onClick={() => props.setDisplay(false)}>X</span>
-        <h1> Error </h1>
-        <p> The item you've entered already exists. </p>
+        <span onClick={() => setDisplay(false)}>X</span>
+        {children}
```

Comment:
Really great idea to use children to make ``Modal`` reusable component. 👏

Reply:
Same! Smart use of it!